cataloguing/additem.pl

```diff
@@ -630,8 +630,15 @@ sub get_common_fields_template {
 
             my $exist_itemnumber;
 
-            for (my $i = 0; $i < $number_of_copies; $i++) {
+            my $total_iterations = 0;
+            my $close_to_infinity = 10_000;
 
+            for (my $i = 0; $i < $number_of_copies; $total_iterations++) {
```

Comment:
`total_iterations` used to prevent the big difference between `i` for the case of the infinite loop (let's say "unknown border case when barcode not generated 10000 times – but against process hanging in such loop).